common/interfaces/user_interface/ui_input_text.c

```diff
@@ -110,6 +110,8 @@ void ui_input_text(const char *input_list,
   ASSERT(initial_heading != NULL);
   ASSERT(max_input_size != 0);
 
+  lv_obj_clean(lv_scr_act());
```

Comment:
do verify if this function (`lv_obj_clean(lv_scr_act())`) call is removed from other places in this file.

Reply:
Removed from 1 destructor callback